README.md

```diff
@@ -2,20 +2,26 @@
 
 The official helpwave web frontends.
 
-# Projects 
+## Projects 
 This repository is split up into multiple subprojects using [pnpm](https://pnpm.io) workspaces.
 - helpwave tasks (see [tasks](/tasks))
 - Landing Page of helpwave (see [landing-page](/landing-page))
 
-# Getting Started
+## Getting Started
 1. `pnpm install`
 1. `cd <project>`
 1. `pnpm run dev`
 1. open [`http://localhost:3000`](http://localhost:3000)
 
-# Testing
+## Storybook
+
+We are about to adopting Storybook to develop, preview, testing and showcasing our components.
+
+- [`lib` https://storybook-lib.helpwave.de](https://storybook-lib.helpwave.de)
+
```

Comment:
Can we add a way to show the storybook locally or add the url to do that here?

Reply:
@DasProffi of course. The ability is already there, I've improved the README.